corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -199,6 +199,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                     $input.val(newValue).trigger('change');
                 }
             });
+            if (this.options.model.get('hidden') === 'true') {
```

Comment:
Minor, but if you put this logic in `query_view.html` and used the `hide` css class, there wouldn't be any chance of a flicker while the page is loading but js hasn't loaded and run this code yet.

Reply:
Since this gets rendered via template from JS, there shouldn't be that issue here?

Reply:
Oh, yes, the template isn't shown until it's rendered, I was thinking of knockout.